### PR TITLE
Metrics doc

### PR DESCRIPTION
**Pull request by Ferril**
# What this PR does

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

**Review by Matvey-Kuk — APPROVED** [review]
I think it worth considering moving "Metrics" to the top because I expect them being more popular & used. Great docs btw!

File: docs/sources/insights-and-metrics/_index.md [hunk 1]
```diff
@@ -6,11 +6,99 @@ keywords:
...
+
+Grafana OnCall Metrics represents certain parameters, such as:
+
+- A total count of alert groups for each integration in every state (firing, acknowledged, resolved, silenced).
+It is a gauge, and its name has the suffix `alert_groups_total`
+- Response time on alert groups for each integration (mean time between the start and first action of all alert groups
+for the last 7 days in selected period). It is a histogram, and its name has the suffix `alert_groups_response_time`
+with the histogram suffixes such as `_bucket`, `_sum` and `_count`
+
+More information about metrics types you can find in the [Prometheus documentation](https://prometheus.io/docs/concepts/metric_types).
```
Comment (matiasb) [inline docs/sources/insights-and-metrics/_index.md:25]:
```suggestion
You can find more information about metrics types in the [Prometheus documentation](https://prometheus.io/docs/concepts/metric_types).
```

File: docs/sources/insights-and-metrics/_index.md [hunk 2]
```diff
@@ -6,11 +6,99 @@ keywords:
...
+
+Metrics have prefix `grafanacloud_oncall_instance`, e.g. `grafanacloud_oncall_instance_alert_groups_total` and
+`grafanacloud_oncall_instance_alert_groups_response_time_seconds_bucket`.
+
+### For open source customers
+
+To collect OnCall application metrics you need to set up Prometheus and add it to your Grafana instance as a datasource.
+You can find more information about Prometheus setup in the [OSS documentation](https://github.com/grafana/oncall#readme)
+
+Metrics have prefix `oncall`, e.g. `oncall_alert_groups_total` and `oncall_alert_groups_response_time_seconds_bucket`.
```
Comment (matiasb) [inline docs/sources/insights-and-metrics/_index.md:42]:
```suggestion
Metrics will have the prefix `oncall`, e.g. `oncall_alert_groups_total` and `oncall_alert_groups_response_time_seconds_bucket`.
```

File: docs/sources/insights-and-metrics/_index.md [hunk 3]
```diff
@@ -6,11 +6,99 @@ keywords:
...
+
+Your metrics also may have additional labels, such as `pod`, `instance`, `container`, depending on your Prometheus setup.
```
Comment (matiasb) [inline docs/sources/insights-and-metrics/_index.md:44]:
```suggestion
Your metrics may also have additional labels, such as `pod`, `instance`, `container`, depending on your Prometheus setup.
```

File: docs/sources/insights-and-metrics/_index.md [hunk 4]
```diff
@@ -6,11 +6,99 @@ keywords:
...
+```promql
+grafanacloud_oncall_instance_alert_groups_response_time_seconds_bucket{slug="test_stack", integration="Grafana Alerting", le="600"}
+```
+
+### Dashboard
+
+To import OnCall metrics dashboard go to `Administration` -> `Plugins` page, find OnCall in the plugins list, open
+`Dashboards` tab at the OnCall plugin settings page and click "Import" near "OnCall metrics". After that you can find
+the "OnCall metrics" dashboard in your dashboards list. In the datasource dropdown select your Prometheus datasource
+(for Cloud customers it's `grafanacluod_usage`). You can filter data by your Grafana instances, teams and integrations.
```
Comment (matiasb) [inline docs/sources/insights-and-metrics/_index.md:94]:
```suggestion
(for Cloud customers it's `grafanacloud_usage`). You can filter data by your Grafana instances, teams and integrations.
```

**Review by matiasb — APPROVED** [review]
LGTM

**Pull request merged**